packages/modelence/src/routes/handler.ts

```diff
@@ -15,11 +16,19 @@ export function createRouteHandler(handler: RouteHandler) {
       try {
         const { session, user } = await authenticate(authToken);
         context = { session, user };
-      } catch (error) {
+      } catch {
         // If authentication fails, context remains null
       }
     }
 
+    const transaction = startTransaction('route', `route:${method.toLowerCase()}:${path}`, {
```

Comment:
Is it scalable to make `path` a part of the name given that it can have many different variations (even for the same route)? I think there are some indexing limits on how many unique values you can use.